prow/cluster/jobs/all-periodics.yaml

```diff
@@ -279,8 +279,58 @@ periodics:
           - entrypoint
           - prow/perf_benchmark_test.sh
 
-- cron: "0 18 * * *"  # starts every day at 18:00PM UTC
-  name: istio-upgrade-helm-chart
+# upgrade using helm from 1.2.8 to 1.3.4
+- cron: "0 2 * * *"  # starts every day at 02:00AM UTC
+  name: istio-upgrade-using-helm
+  branches: master         # this master branch is for prow/upgrade-tests.sh
+  decorate: true
+  extra_refs:
+    - org: istio
+      repo: istio
+      base_ref: master
+      path_alias: istio.io/istio
+  annotations:
+    testgrid-dashboards: istio_release-pipeline
+    testgrid-alert-email: istio-oncall@googlegroups.com
+    testgrid-num-failures-to-alert: '1'
+  labels:
+    preset-service-account: "true"
+  spec:
+    containers:
+      - <<: *istio_container_with_kind
+        env:
+          - name: SOURCE_TAG
```

Comment:
I think we should just always set TARGET_TAG, to be more explicit

Reply:
ok, I see, I will add TARGET_TAG for every job